### DISCUSSION TITLE
USE STM32 F411CE

### DISCUSSION THREAD

**Discussion by XiaoBaiXJ** [nanoframework/Home#1556] [Q&A]
### Location

Docs website

### Description

![image](https://github.com/user-attachments/assets/29767353-b7dd-42a7-8ffd-b6595f2d9066)
I have already burned the firmware and connected it using a virtual serial port and USB. The virtual serial port keeps going online and offline again. When I checked with an oscilloscope, I found that the voltage was always 3.3V

### Screenshots

How should this be resolved

**Comment by XiaoBaiXJ** [comment]
![image](https://github.com/user-attachments/assets/c7b73065-6071-4957-bcd1-e2f775885fbe)
This is all firmware information

**Reply by josesimoes** [reply]
What is the board you have? If it is an ST Kit you should be using one of the ST Discovery or Nucleo for  the 411, not the WEACT.

**Comment by XiaoBaiXJ** [comment]
If using a serial port for connection, does it require TTL conversion? How to configure it

**Comment by XiaoBaiXJ** [comment]
Do I need to use two power cords to connect the microcontroller when installing with DFU? I can use DFU to burn the framework by short circuiting 3.3V, but I cannot connect it using VS2022. Why is this?

**Comment by josesimoes** [comment]
This is a question not an issue. Transfering.
